components/hab/src/cli.rs

```diff
@@ -403,6 +404,35 @@ pub fn get(feature_flags: FeatureFlag) -> App<'static, 'static> {
                      (@arg AUTH_TOKEN: -z --auth +takes_value "Authentication token for Builder")
                 )
             )
+            (@subcommand rbac=>
+                (about: "Role Based Access Control for origin members")
+                (@setting ArgRequiredElseHelp)
+                (@setting SubcommandRequiredElseHelp)
+                (@subcommand show =>
+                     (about: "Display an origin member's current role")
+                     (@arg MEMBER_ACCOUNT: +required +takes_value {non_empty} "The account name of the role to display")
+                     (@arg ORIGIN: -o --origin +takes_value +required {valid_origin} "The name of the origin for the origin member association")
+                     (@arg BLDR_URL: -u --url +takes_value {valid_url}
+                          "Specify an alternate Builder endpoint. If not specified, the value will \
+                          be taken from the HAB_BLDR_URL environment variable if defined. (default: \
+                          https://bldr.habitat.sh)")
+                     (@arg AUTH_TOKEN: -z --auth +takes_value "Authentication token for Builder")
+                     (@arg TO_JSON: -j --json "Output will be rendered in json")
+                )
+                (@subcommand set =>
+                     (about: "Change an origin member's role")
+                     (@arg MEMBER_ACCOUNT: +required +takes_value {non_empty} "The account name whose role will be changed")
+                     (@arg ORIGIN: -o --origin +takes_value +required {valid_origin} "The name of the origin for the origin member association")
+                     (@arg ROLE: --role -r +takes_value +required {valid_role} "The role name to enforce for the member account \
+                          [values: member, maintainer, administrator]")
+                     (@arg BLDR_URL: -u --url +takes_value {valid_url}
+                          "Specify an alternate Builder endpoint. If not specified, the value will \
+                          be taken from the HAB_BLDR_URL environment variable if defined. (default: \
+                          https://bldr.habitat.sh)")
+                     (@arg AUTH_TOKEN: -z --auth +takes_value "Authentication token for Builder")
+                     (@arg NO_PROMPT: -n --("no-prompt") "Do not prompt for confirmation")
+                )
+            )
```

Comment:
Because this is a new command I think we should go all in on `structopt` and use it instead of the explicit `clap` version. To keep the `structopt` and `clap` versions in sync we still want to add the subcommand, but we can do that with `(subcommand Rbac::clap())`. See [here](https://github.com/habitat-sh/habitat/blob/9472ea6f36ff1d2aaba6be16a736a60ab8d4ad1e/components/hab/src/cli.rs#L881) for an example.